.github/workflows/go.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         curl -Ls "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash
         mv kustomize ./bin/
 
-        export PATH=$PATH:$(pwd)/bin
+        export PATH=$(pwd -P)/bin:$PATH
```

Comment:
What was the rationable behind this change? 🤔 
I think this can prevent it from preferring kustomize binaries installed into `$(pwd)/bin`.

Reply:
@mumoshu The script (line 30th) installs the binary into `$(pwd)/bin`. The new `export` change is to ensure that new binary is picked up in the subsequent executions. 

On my laptop I use an old version of `kustomize` (3.4) and it took me a while to recognize that I needed to install the latest `kustomize` binary and adjusted the path as above. That change is also required if the building context/container already has any old version of kustomization.

(`pwd -P` will use the physical path; it's the same output as `pwd` in our case.)

This is not a big change, I can help to remove it so the primary idea of the fix matters.

Thanks.

Reply:
Ahh, gotcha. Sorry I somehow read it as the opposite. This will be required to make the installed `kustomize` binary preferred than the one that is installed system-wide. Thanks for clarifying!